fireant/slicer/transformers/highcharts.py

```diff
@@ -109,7 +136,7 @@ def yaxis_options(self, dataframe, dim_ordinal, display_schema):
         reference_axes_ids = set(self._reference_axes_id(reference_key) for reference_key in reference_keys)
 
         for axes_id in reference_axes_ids:
-            axis.append({'id': axes_id, 'title': {'text': None}})
+            axis.append({'id': axes_id, 'title': {'text': 'References'}, 'opposite': True})
```

Comment:
could you make only the first reference axis use the title text? Otherwise it's going to take up more space